tests/spawn-after-drop.rs

```diff
@@ -0,0 +1,52 @@
+use std::{env, process::Command};
+
+use jobserver::{Client, FromEnvErrorKind};
+
+macro_rules! t {
```

Comment:
I don't think we need `t!` anymore, it's probably used in times where `.unwrap()` isn't available